src/components/HeaderComponent/HeaderComponent.jsx

```diff
@@ -34,7 +35,7 @@ const HeaderComponent = () => {
     <header className="headerNavBar">
       {showLogIn ? <LogInComponent setShowLogIn={setShowLogIn} /> : ""}
```

Comment:
I suggest this:
`{showLogIn ? <LogInComponent setShowLogIn={setShowLogIn} /> : <></>}`

Reply:
Resolved